products/sqlite.md

```diff
@@ -0,0 +1,333 @@
+    latest: "3.22.0"
+    latestReleaseDate: 2018-01-22
+    releaseDate: 2018-01-22
+
+-   releaseCycle: "3.21"
+    eol: false
+    latest: "3.21.0"
+    latestReleaseDate: 2017-10-24
+    releaseDate: 2017-10-24
+
+-   releaseCycle: "3.20"
+    eol: false
+    latest: "3.20.1"
+    latestReleaseDate: 2017-08-24
+    releaseDate: 2017-08-01
+
+-   releaseCycle: "3.19"
+    eol: false
+    latest: "3.19.3"
+    latestReleaseDate: 2017-06-08
+    releaseDate: 2017-05-22
+
+-   releaseCycle: "3.18"
+    eol: false
+    latest: "3.18.2"
+    latestReleaseDate: 2017-06-17
+    releaseDate: 2017-03-28
+
+-   releaseCycle: "3.17"
+    eol: false
+    latest: "3.17.0"
+    latestReleaseDate: 2017-02-13
+    releaseDate: 2017-02-13
+
+-   releaseCycle: "3.16"
+    eol: false
+    latest: "3.16.2"
+    latestReleaseDate: 2017-01-06
+    releaseDate: 2017-01-02
+
+-   releaseCycle: "3.15"
+    eol: false
+    latest: "3.15.2"
+    latestReleaseDate: 2016-11-28
+    releaseDate: 2016-10-14
+
+-   releaseCycle: "3.14"
+    eol: false
+    latest: "3.14.2"
+    latestReleaseDate: 2016-09-12
+    releaseDate: 2016-08-08
+
+-   releaseCycle: "3.13"
+    eol: false
+    latest: "3.13.0"
+    latestReleaseDate: 2016-05-18
+    releaseDate: 2016-05-18
+
+-   releaseCycle: "3.12"
+    eol: false
+    latest: "3.12.2"
+    latestReleaseDate: 2016-04-18
+    releaseDate: 2016-03-29
+
+-   releaseCycle: "3.11"
+    eol: false
+    latest: "3.11.1"
+    latestReleaseDate: 2016-03-03
+    releaseDate: 2016-02-15
+
+-   releaseCycle: "3.10"
+    eol: false
+    latest: "3.10.2"
+    latestReleaseDate: 2016-01-20
+    releaseDate: 2016-01-06
+
+-   releaseCycle: "3.09"
+    eol: false
+    latest: "3.09.02"
+    latestReleaseDate: 2015-11-02
+    releaseDate: 2015-10-14
+
+-   releaseCycle: "3.08"
+    eol: false
+    latest: "3.08.11.01"
+    latestReleaseDate: 2015-07-29
+    releaseDate: 2013-08-26
+
+-   releaseCycle: "3.07"
+    eol: false
+    latest: "3.07.17"
+    latestReleaseDate: 2013-05-20
+    releaseDate: 2010-07-21
+
+-   releaseCycle: "3.06"
+    eol: false
+    latest: "3.06.23.01"
+    latestReleaseDate: 2010-03-26
+    releaseDate: 2008-07-16
+
+-   releaseCycle: "3.05"
+    eol: false
+    latest: "3.05.09"
+    latestReleaseDate: 2008-05-14
+    releaseDate: 2007-09-04
+
+-   releaseCycle: "3.04"
+    eol: false
+    latest: "3.04.02"
+    latestReleaseDate: 2007-08-13
+    releaseDate: 2007-06-18
+
+-   releaseCycle: "3.03"
+    eol: false
+    latest: "3.03.17"
+    latestReleaseDate: 2007-04-25
+    releaseDate: 2006-01-11
+
+-   releaseCycle: "3.02"
+    eol: false
+    latest: "3.02.08"
+    latestReleaseDate: 2005-12-19
+    releaseDate: 2005-03-21
+
+-   releaseCycle: "3.01"
+    eol: false
+    latest: "3.01.06"
+    latestReleaseDate: 2005-03-17
+    releaseDate: 2005-01-21
+
+-   releaseCycle: "3.0"
+    eol: false
+    latest: "3.0.8"
+    latestReleaseDate: 2004-10-12
+    releaseDate: 2004-06-18
+
+-   releaseCycle: "2.08"
```

Comment:
Are multiple minor versions of the same major supported in parallel ? If no, `2` cycles should be collapsed.

Reply:
According to their [LTS Policies](https://www.sqlite.org/lts.html), I may say that they do : do you share the same analysis ?

Reply:
I do not have the same opinion about the LTS policies. I they are just saying they plan to develop SQLite as if the project will be supported until 2050.

You should be looking at https://sqlite.org/com/tech.html :

>  We will support any version of SQLite up to a year old, **not just the latest version**. At your request, we will backport any bug fixes to any SQLite version that is less than one year old.

Reply:
Versions collapsed and description updated.